multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -65,9 +71,20 @@ type MemberClusterAnnounceReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
 
-	mapLock      sync.Mutex
-	memberStatus map[common.ClusterID]*multiclusterv1alpha1.ClusterStatus
-	timerData    map[common.ClusterID]*timerData
+	mapLock                        sync.Mutex
+	memberStatus                   map[common.ClusterID]*multiclusterv1alpha1.ClusterStatus
+	timerData                      map[common.ClusterID]*timerData
+	installedMemberClusterAnnounce cache.Indexer
```

Comment:
As we have only a single index and we already used map in the same struct, I would suggest to use map instead.